### ISSUE TITLE
An error with nn.CrossEntropyLoss()

### ISSUE THREAD

**Issue by Perkzi** [FedPruning/FedPruning#22]
An error is reported
in api/standalone/fedtinyclean/my_model_trainer_classification.py

I am not sure if it is my problem only

Maybe nn.CrossEntropyLoss() needs to turn labels to long tensor. 
I add labels = labels.long() every time before using nn.CrossEntropyLoss(), and it becomes ok.
